HeterogeneousCore/SonicTriton/scripts/triton

```diff
@@ -0,0 +1,303 @@
+#!/bin/bash
+
+# defaults
+USEDOCKER=""
+GPU=""
+VERBOSE=""
+WTIME=60
+SERVER=triton_server_instance
+RETRIES=3
+REPOS=()
+MODELS=()
+FORCE=""
+CLEANUP=true
+TMPDIR=""
+DRYRUN=""
+
+usage() {
+	ECHO="echo -e"
+	$ECHO "triton [options] [start|stop]"
```

Comment:
I don't mind renaming it, but would prefer to rename it at most once (needs to propagate to several places), so let's settle on a name first. `cmsTriton` is fine with me. (While it's currently used for launching "local" servers, it might eventually be used by grid sites to launch their "remote" servers also.)